PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsHostConfigurationViewModel.kt

```diff
@@ -120,45 +136,49 @@ class AzureFunctionsHostConfigurationViewModel(
                 }
     }
 
+    // TODO: FIX_WHEN. Add integration tests when enabled in the plugin.
     private fun recalculateTrackProjectOutput() {
-        val selectedProject = projectSelector.project.valueOrNull
-        val selectedTfm = tfmSelector.string.valueOrNull
-        if (selectedProject != null && selectedTfm != null) {
-            selectedProject.projectOutputs.singleOrNull { it.tfm == selectedTfm }?.let {
-                trackProjectExePath = exePathSelector.path.value == it.exePath
-                trackProjectArguments = (it.defaultArguments.isEmpty()
-                        || programParametersEditor.parametersString.value ==
-                        ParametersListUtil.join(it.defaultArguments))
-                trackProjectWorkingDirectory = workingDirectorySelector.path.value == it.workingDirectory
-            }
+        val selectedProject = projectSelector.project.valueOrNull ?: return
+        val selectedTfm = tfmSelector.string.valueOrNull ?: return
 
-            val result = portRegex.find(programParametersEditor.parametersString.value)
-            if (result != null && result.groups.count() == 2) {
-                urlEditor.defaultValue.value = "http://localhost:${result.groupValues[1]}"
-                urlEditor.text.value = "http://localhost:${result.groupValues[1]}"
-                dotNetBrowserSettingsEditor.settings.value = BrowserSettings(false, false, null)
```

Comment:
@maartenba, I did not get the idea to have this hardcoded here. Please let me know if I miss anything in the initial implementation.

Reply:
Seems good!